web/lib/litegraph.core.js

```diff
@@ -7559,11 +7559,11 @@ LGraphNode.prototype.executeAction = function(action)
         this.ds.offset[0] =
             -node.pos[0] -
             node.size[0] * 0.5 +
-            (this.canvas.width * 0.5) / this.ds.scale;
+            (this.canvas.width * 0.5) / (this.ds.scale * window.devicePixelRatio);
```

Comment:
Shouldn't this go to upstream litegraph repo?
https://github.com/comfyanonymous/litegraph.js

Reply:
Current litegraph used in the repo is hosted on https://github.com/comfyanonymous/litegraph.js/tree/comfyui branch

Reply:
Sure, thanks